Source/ServerTrustEvaluation.swift

```diff
@@ -396,3 +518,7 @@ public extension SecCertificate {
         return SecTrustCopyPublicKey(createdTrust)
     }
 }
+
+extension OSStatus {
+    var isSuccess: Bool { return self == errSecSuccess }
+}
```

Comment:
Should we consider making all these extensions `public` for all those out there that would like to make their own custom evaluators? I certainly like the flexibility to control these APIs internally, but these are fantastic convenience functions for everyone.

Reply:
Indeed. They would certainly need a namespace of some sort. I lean towards the `.af` property. Not something we'd need to do in this PR, just wanted to ask.